urwid/event_loop/asyncio_loop.py

```diff
@@ -73,6 +81,10 @@ def __init__(self, *, loop: asyncio.AbstractEventLoop | None = None, **kwargs) -
         self._idle_handle: int = 0
         self._idle_callbacks: dict[int, Callable[[], typing.Any]] = {}
 
+    def __del__(self) -> None:
+        if self.__event_loop_policy_altered:
+            asyncio.set_event_loop_policy(None)  # Restore default event loop policy
```

Comment:
should we capture the previous policy and restore it instead of setting back to the default?

Reply:
done